src/styles/blocks/header.scss

```
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: all 300ms;
```

Specify the transition property for readability:
```suggestion
    transition: color 300ms;
```